components/ingest-service/backend/client.go

```diff
@@ -72,8 +78,10 @@ type Client interface {
 	JobStatus(context.Context, string) (project_update_lib.JobStatus, error)
 	// @param (context, jobID)
 	JobCancel(context.Context, string) error
-
+	// @param (context, projectRules)
 	UpdateProjectTags(context.Context, map[string]*iam_v2.ProjectRules) ([]string, error)
+	// @param (context, projectTagToDelete)
+	DeleteProjectTag(context.Context, string) ([]string, error)
```

Comment:
I can do this in the follow-up PR for wiring things together, but need to adjust the signature to this:
```suggestion
	PurgeProject(context.Context, string) error
```

Reply:
I think we will need the `[]string` return because that is the job IDs of the running task in Elasticsearch. We can use these IDs to check when the job is complete or for errors.

Reply:
Thanks for that pointer. In my PR I will encapsulate that check within `PurgeProject` itself. TL;DR can merge yours as is and my PR will make the necessary modifications. Thanks!